.github/workflows/app-publish.yaml

```
@@ -50,7 +50,7 @@ jobs:
       - name: Upload to repo.jellyfin.org
         uses: burnett01/rsync-deployments@4.1
         with:
-          switches: -rltgoDzvO --delete --exclude='*' --include='**/*.apk' --include='*.txt'
+          switches: -avzrtO --delete
```

```suggestion
          switches: -avz
```

- `-a` implies `-rt`
- Why the `-O`? Shouldn't be needed
- `--delete` might be unsafe and isn't necessary - if we retag a release, the filenames shouldn't change
- On the `-a`: do we really need to preserve owner, group, ...? `-rpt` might be enough (ignoring owner, group, links and device/special files).

---

I could have sworn that I read a does not imply `r`which is why its there

---

I agree with removing `--delete`. This could have dangerous potential implications based on the directory. The collection task should *move* (`mv`) the files to their final locations thus removing them and leaving the directory empty anyways.

---

`-a` does imply `-r` ;-)

---

> I could have sworn that I read `-a` does not imply `-r`which is why its there

[`-a` is equivalent to that neat string `-rlptgoD`](https://linux.die.net/man/1/rsync). *I also had to look it up earlier.*